src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -681,20 +682,21 @@ describe('ReactUpdates', () => {
       }
 
       componentDidMount() {
+        this.container = document.createElement('div');
```

Comment:
Specifically we want to test that we can render into the node we already rendered.